packages/dds/ink/src/inkCanvas.ts

```diff
@@ -46,15 +46,21 @@ function drawPolygon(context: CanvasRenderingContext2D, points: IPoint[]) {
 
 	context.beginPath();
 	// Move to the first point
-	context.moveTo(points[0].x, points[0].y);
+	// Non null asserting, this must exist because the length is not 0
```

Comment:
length check is not sufficient to ensure an entry is not undefined.
Have a read thru https://typescript.tv/best-practices/safer-array-access-in-typescript/.
Per instance, we need to decide how defensive we need to be with each case. If a customer sends us questionable array, when do need to be careful or when should we let things blow up? I think for our purposes we don't care so much.
The main interest is noted here https://www.semver-ts.org/formal-spec/5-compiler-considerations.html#strictness.

Reply:
I learned something new, but it makes sense, you can make an array of size without filling it, crazy